setup.py

```diff
@@ -185,6 +185,7 @@
     "dicom": ["pydicom>=2.0.0", "pynetdicom", "pylibjpeg-libjpeg"],
     "docs": ["jupyter-book>=0.8.3", "sphinx-argparse", "sphinxcontrib-napoleon"],
     "doctests": ["tensorflow>=2.2.0", "black>=19.3b0,<20.0", "tomlkit"],
+    "external": ["flashgamma"],
```

Comment:
external is a very broad term.
in the property  (land/housing) world, this kind of thing is referred to as "comparables" or "comps".

Reply:
> external is a very broad term.

Yeah, good call.

> in the property (land/housing) world, this kind of thing is referred to as "comparables" or "comps".

I like it :) Will make the change.